lib/google_drive/worksheet.rb

```diff
@@ -139,6 +139,19 @@ def human_url
       format("%s\#gid=%s", spreadsheet.human_url, gid)
     end
 
+    # copy worksheet to specified spreadsheet
+    def copy_to(spreadsheet_id)
+      request = Google::Apis::SheetsV4::CopySheetToAnotherSpreadsheetRequest.new(
+        destination_spreadsheet_id: spreadsheet_id,
+      )
+      @session.sheets_service.copy_spreadsheet(spreadsheet_id, sheet_id, request)
+    end
```

Comment:
Can you explicitly return nil from this method for now?

It would be better to return GoogleDrive::Worksheet instance of the created copy, but it may be tricky. It would be less confusing to return nil until then.